### PR TITLE
Update readme for NERD Commenter

### LINKED ISSUE #97: Suggestion: Change tinycomment for tcomment or smth else
This is just a suggestion to change `tinycomment` for something else. `tcomment` comes to mind since it's still maintained and tinycomment has seen it's last commit 2 years ago.

btw, `<leader>-v` for commenting is also not working out of the box

### PR DESCRIPTION
**Pull request by wsdjeg**
Fix #97

### CODE REVIEW

**Review by cserb — APPROVED** [review]
Everything works for me except `<leader>+cs` in that it destroys the formatting. But this looks more like a nerd commenter problem together with ruby

**Comment by wsdjeg** [comment]
Open issue in it's repo maybe better, and I think should provide the diff  before comment and after it. 

so can we merge this PR?

File: config/plugins/vimfiler.vim [hunk 1]
```diff
@@ -57,6 +57,7 @@ function! s:vimfilerinit()
     silent! nunmap <buffer> gr
     silent! nunmap <buffer> gf
     silent! nunmap <buffer> -
+    silent! nunmap <buffer> s
```
Comment (cserb) [inline config/plugins/vimfiler.vim:60]: not sure what this line has to do with nerd commenter

**Review by cserb — APPROVED** [review]
yes the problem described has nothing to with this PR and was just as a fyi (maybe wrong place)

we are good to go

File: config/plugins/vimfiler.vim (hunk 1 above)
Comment (wsdjeg) [reply config/plugins/vimfiler.vim:60]: Because I use s as my windows func leader, so I need disable it in vimfiler buffer.

Comment (cserb) [reply config/plugins/vimfiler.vim:60]: 👍

Comment (wsdjeg) [reply config/plugins/vimfiler.vim:60]: thanks, merged.

**Pull request merged**